pages/in/embed/_id/index.vue

```diff
@@ -111,6 +111,12 @@ export default {
         };
       })
       .catch(() => {});
+
+    data.platforms = await apiGetSocialListById(params.id)
```

Comment:
probably should user `Promise.All()` for the 2 api, each with its own `catch`?

Reply:
Updated